playbooks/roles/cluster/templates/cluster_provision_survey.json.j2

```diff
@@ -27,7 +27,7 @@
       {
       "question_description": "Flag for provisioning logging stack",
       "min": null,
-      "default": "true",
+      "default": "false",
```

Comment:
Given that our POC tower instance is not updated on a regular basis we should be ok here. We also explicitly state that it should be `true` in our POC provisioning [SOP](https://github.com/RHCloudServices/integreatly-help/blob/master/sops/POC_provisioning.asciidoc#32-run-cluster-provision-workflow)